tools/shared.py

```diff
@@ -809,8 +809,8 @@ def expand_byte_size_suffixes(value):
   if not match:
     try:
       return int(value)
-    except ValueError:
-      raise Exception("Invalid byte size, valid suffixes: KB, MB, GB, TB")
+    except ValueError as e:
```

Comment:
`e` is not used?

Reply:
Ops, that was an accident when debugging, reverted.